<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QWalkingMapsClass</class>
 <widget class="QMainWindow" name="QWalkingMapsClass">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>面向对象的步行地图导航</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout" stretch="1,10">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,4,1,4">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>起点</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="CompleteLineEdit" name="FromSearchLine"/>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>终点</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="CompleteLineEdit" name="ToSearchLine"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="MyView" name="graphicsView"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyView</class>
   <extends>QGraphicsView</extends>
   <header>MyView.h</header>
  </customwidget>
  <customwidget>
   <class>CompleteLineEdit</class>
   <extends>QLineEdit</extends>
   <header>CompleteLineEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="QWalkingMaps.qrc"/>
 </resources>
 <connections/>
</ui>
